src/nyc_trees/apps/event/models.py

```diff
@@ -100,6 +109,21 @@ class EventRegistration(NycModel, models.Model):
     class Meta:
         unique_together = ('user', 'event')
 
+    @classmethod
+    def my_events_now(cls, user):
+        """
+        Return events for which user has RSVPd but not checked in,
+        with times:  start-1h < now < end
+        """
+        now = timezone.now()
+        registrations = EventRegistration.objects \
+            .filter(user=user,
+                    did_attend=False,
+                    event__begins_at__lte=now + STARTING_SOON_WINDOW,
+                    event__ends_at__gt=now) \
+            .prefetch_related('event')
+        return [r.event for r in registrations]
```

Comment:
Do we want to show an event alert for group admins? Since they do not have RSVPs for their own events this will always return an empty list for group admins.